polaris/ocean/rpe.py

```diff
@@ -5,38 +5,42 @@
 from mpas_tools.cime.constants import constants
 
 
-def compute_rpe(initial_state_file_name, output_files):
+def compute_rpe(mesh_filename, initial_state_filename, output_filenames):
     """
     Computes the reference (resting) potential energy for the whole domain
 
     Parameters
     ----------
-    initial_state_file_name : str
+    mesh_filename : str
+        Name of the netCDF file containing the MPAS horizontal mesh variables
+
+    initial_state_filename : str
         Name of the netCDF file containing the initial state
 
-    output_files : list
+    output_filenames : list
         List of netCDF files containing output of forward steps
 
     Returns
     -------
     rpe : numpy.ndarray
         the reference potential energy of size ``Time`` x ``len(output_files)``
     """
-    num_files = len(output_files)
+    num_files = len(output_filenames)
     if num_files == 0:
         raise ValueError('Must provide at least one output filename')
 
     gravity = constants['SHR_CONST_G']
 
-    dsInit = xr.open_dataset(initial_state_file_name)
-    nVertLevels = dsInit.sizes['nVertLevels']
+    ds_mesh = xr.open_dataset(mesh_filename)
+    ds_init = xr.open_dataset(initial_state_filename)
+    nVertLevels = ds_init.sizes['nVertLevels']
 
-    xEdge = dsInit.xEdge
-    yEdge = dsInit.yEdge
-    areaCell = dsInit.areaCell
-    minLevelCell = dsInit.minLevelCell - 1
-    maxLevelCell = dsInit.maxLevelCell - 1
-    bottomDepth = dsInit.bottomDepth
+    xEdge = ds_mesh.xEdge
+    yEdge = ds_mesh.yEdge
+    areaCell = ds_mesh.areaCell
```

Comment:
This change is in the spirit of not assuming that mesh variables are in the initial condition.